### DISCUSSION TITLE
Utilisation du paramètre "need_reload"

### DISCUSSION THREAD

**Discussion by adipsostoklosa** [PrestaSafe/prettyblocks#232] [Q&A] [answered]
Bonjour, je me penche sur le paramètre "need_reload". J'ai bien compris que, lorsqu'il est à true, c'est l'iframe entier qui se recharge. Mais j'ai surtout remarqué que lorsque la valeur est à false, on a un rendu plus rapide de nos modifications. Alors dans quels cas concrets est-il plus adapté d'avoir need_reload à true ?
Merci d'avance pour vos réponses

**Comment by PrestaSafe** [comment, marked as answer]
Bonjour, c'est bien le cas. 

Le but est pour les blocks de type slider qui ont besoin de recharger leur propre javascript, à ce jour, nous n'émettons pas d'evenements sur l'iframe pour que le block puisse se recharger automatiquement nous le ferons dans les futurs version. 
Cordialement
